client/app/ip/ip-lb/ip-ip-lb.controller.js

```diff
@@ -1,11 +1,12 @@
-angular.module('Module.ip.controllers').controller('IplbDashboardCtrl', ($scope, $rootScope, $q, $location, $translate, Iplb, Alerter) => {
+angular.module('Module.ip.controllers').controller('IplbDashboardCtrl', ['$location', '$q', '$rootScope', '$scope', '$translate', 'Alerter', 'Iplb', 'UserAccount.services.Contacts', ($location, $q, $rootScope, $scope, $translate, Alerter, Iplb, Contacts) => {
```

Comment:
Isn't `ngAnnotate` available anymore ?

Reply:
I was not able to annotate 'UserAccount.services.Contacts' and hence I remove ngAnnotate usage here